src/taipy/core/pipeline/_pipeline_manager.py

```diff
@@ -128,6 +128,12 @@ def __save_tasks(cls, tasks):
         for i in tasks:
             task_manager._set(i)
 
+    @classmethod
+    def _is_submittable(cls, pipeline: Union[Pipeline, PipelineId]) -> bool:
+        if isinstance(pipeline, str):
+            pipeline = cls._get(pipeline)
+        return True if isinstance(pipeline, Pipeline) else False
```

Comment:
```suggestion
        return isinstance(pipeline, Pipeline)
```
This should be enough, right?